tests/testthat/test-loop-over-all-stages-logging.R

```diff
@@ -1,4 +1,4 @@
-skip()
+skip("Emil should look at these errors")
```

Comment:
The snapshot diffs look good to me. There are some non-spanshot errors that need some looking at where the differences are related to the `trace` column/object